docs/API_spec.yml

```diff
@@ -0,0 +1,190 @@
+openapi: "3.0.2"
```

Comment:
all the OpenAPIs files can be merged into one and grouped with tags https://swagger.io/docs/specification/grouping-operations-with-tags/